net/nextdns/Makefile

```diff
@@ -50,13 +50,14 @@ define Package/nextdns/install
 
 	$(INSTALL_DIR) $(1)/etc/config
 	$(INSTALL_CONF) ./files/nextdns.config $(1)/etc/config/nextdns
+
+	$(INSTALL_DIR) $(1)/etc/uci-defaults
+	$(INSTALL_BIN) ./files/nextdns.defaults $(1)/etc/uci-defaults/nextdns
 endef
 
 define Package/nextdns/postinst
 #!/bin/sh
-if [ -z "$${IPKG_INSTROOT}" ]; then
-  nextdns install
-fi
+[ -n "$${IPKG_INSTROOT}" ] || (. /etc/uci-defaults/nextdns) && rm -f /etc/uci-defaults/nextdns
```

Comment:
That logic is already covered by the [default postinstall procedure](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/functions.sh#L284), so you can remove the `postinst` section entirely from the Makefile.